scripts/build-embedded

```diff
@@ -21,7 +21,7 @@ DIR=${GIT_TAG:-$COMMIT_BRANCH}
 OUTPUT_DIR=dist/${DIR}-embedded
 
 echo "Building..."
-NUXT_ENV_commit=${COMMIT} NUXT_ENV_version=${VERSION} OUTPUT_DIR=$OUTPUT_DIR ROUTER_BASE='/dashboard' yarn run build
+COMMIT=${COMMIT} VERSION=${VERSION} OUTPUT_DIR=$OUTPUT_DIR ROUTER_BASE='/dashboard' yarn run build
```

Comment:
For reference this is the PR https://github.com/rancher/dashboard/pull/7516

Reply:
This is the part that adds the version env variable https://github.com/rancher/dashboard/blob/92a6911b1aa3c7b6b5d031bb8f87649d3af89c1f/scripts/build-embedded#L16

I did have to make changes to the vue.config.js since it doesn't have a concept of publicRuntimeConfig. I defined that using definePlugin and updated the nuxt/client.js since definePlugin didn't seem to support replacing the NUXT prefix.